internal/monitor/instance/docker/monitor.go

```diff
@@ -180,6 +193,7 @@ func DefaultConfig() *Config {
 		SocketAddress:              constants.DefaultDockerSocket,
 		SyncAtStart:                true,
 		KillContainerOnPolicyError: false,
+		ECS: false,
```

Comment:
See above. Nothing is ECS specific here. It is just a different mode.

Reply:
Renamed ECS to NoProxyMode